tests/profiles/athena/test_athena_access_key.py

```diff
@@ -136,8 +197,8 @@ def test_athena_profile_args_overrides(
     assert profile_mapping.profile == {
         "type": "athena",
         "aws_access_key_id": mock_athena_conn.login,
-        "aws_secret_access_key": "{{ env_var('COSMOS_CONN_AWS_AWS_SECRET_ACCESS_KEY') }}",
-        "aws_session_token": "{{ env_var('COSMOS_CONN_AWS_AWS_SESSION_TOKEN') }}",
+        "aws_secret_access_key": mock_assumed_credentials.secret_key,
+        "aws_session_token": mock_assumed_credentials.token,
```

Comment:
Before, we were obfuscating the secrets and setting them in environment variables. This behaviour should continue for temporary credentials as well.

Reply:
Great call out, sorry I missed it. Ive fixed it up now. Let me know what you think of the solution.